ivy/container/set.py

```diff
@@ -4,12 +4,47 @@
 # local
 import ivy
 from ivy.container.base import ContainerBase
+import ivy
+from typing import Optional, List, Union, Dict
```

Comment:
This import has been already made in line 2, please update that line as redefining fails the lint tests